### DISCUSSION TITLE
App Execution Time

### DISCUSSION THREAD

**Discussion by anasshamoon12002** [arc53/DocsGPT#164] [Q&A] [answered]
I am running the app locally on mac but it takes like 15 - 40 secs to generate the response. Can you please tell me what's going on and what is the solution?

**Comment by dartpain** [comment, marked as answer]
Its quering your data in the background, its not like just one question and response with chatgpt, but multiple questions that happend at the same time. Where results are combined in the end

**Reply by anasshamoon12002** [reply]
The app deployed is way faster so how does it generate the answers fast? Is it something related to the processing power or GPU support?
